src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -756,6 +758,86 @@ public async Task SendUpdatedTempPasswordEmailAsync(string email, string userNam
             await _mailDeliveryService.SendEmailAsync(message);
         }
 
+        public async Task SendFamiliesForEnterpriseOfferEmailAsync(string email, string sponsorEmail, bool existingAccount, string token)
+        {
+            var message = CreateDefaultMessage("Accept Your Free Families Subscription", email);
+
+            if (existingAccount)
+            {
+                var model = new FamiliesForEnterpriseOfferExistingAccountViewModel
+                {
+                    SponsorEmail = sponsorEmail,
+                    SponsoredEmail = WebUtility.UrlEncode(email),
+                    WebVaultUrl = _globalSettings.BaseServiceUri.VaultWithHash,
+                    SiteName = _globalSettings.SiteName,
+                    SponsorshipToken = token,
+                };
+
+                await AddMessageContentAsync(message, "FamiliesForEnterprise.FamiliesForEnterpriseOfferExistingAccount", model);
+            }
+            else
+            {
+                var model = new FamiliesForEnterpriseOfferNewAccountViewModel
+                {
+                    SponsorEmail = sponsorEmail,
+                    SponsoredEmail = WebUtility.UrlEncode(email),
+                    WebVaultUrl = _globalSettings.BaseServiceUri.VaultWithHash,
+                    SiteName = _globalSettings.SiteName,
+                    SponsorshipToken = token,
+                };
+
+                await AddMessageContentAsync(message, "FamiliesForEnterprise.FamiliesForEnterpriseOfferNewAccount", model);
+            }
+
+            message.Category = "FamiliesForEnterpriseOffer";
+            await _mailDeliveryService.SendEmailAsync(message);
+        }
+
+        public async Task SendFamiliesForEnterpriseRedeemedEmailsAsync(string familyUserEmail, string sponsorEmail)
+        {
+            // Email family user
+            await SendFamiliesForEnterpriseInviteRedeemedToFamilyUserEmailAsync(familyUserEmail);
+
+            // Email enterprise org user
+            await SendFamiliesForEnterpriseInviteRedeemedToEnterpriseUserEmailAsync(sponsorEmail);
+        }
+
+        private async Task SendFamiliesForEnterpriseInviteRedeemedToFamilyUserEmailAsync(string email)
+        {
+            var message = CreateDefaultMessage("Success! Families Subscription Accepted", email);
+            var model = new FamiliesForEnterpriseRedeemedToFamilyUserViewModel
+            {
+
+            };
```

Comment:
I know this model is empty, just pointing out again that it seems like we aren't really using it for anything.